definitions/npm/prop-types_v15.x.x/flow_v0.89.x-/prop-types_v15.x.x.js

```diff
@@ -0,0 +1,34 @@
+type $npm$propTypes$ReactPropsCheckType = (
+  props: any,
+  propName: string,
+  componentName: string,
+  href?: string
+) => ?Error;
+
+// Copied from: https://github.com/facebook/flow/blob/0938da8d7293d0077fbe95c3a3e0eebadb57b012/lib/react.js#L433-L449
+declare module 'prop-types' {
+  declare var array: React$PropType$Primitive<Array<any>>;
+  declare var bool: React$PropType$Primitive<boolean>;
+  declare var func: React$PropType$Primitive<Function>;
```

Comment:
I know it's copied from the React sources, but `Function`, `Object` is any.

Reply:
I know too) 
What do you suggest I need to change?

Reply:
Always the same dilemma... No perfect solutions I'm afraid. `() => void` or `() => {}` for `Function`, `{ +[string]: mixed }` for `Object`...